packages/web-react/src/components/Modal/README.md

```diff
@@ -301,15 +300,19 @@ takes over the responsibility for scrolling and provides visual overflow decorat
 By default, ModalDialog grows to the height of its content until it reaches the [maximum height](#custom-max-height)
 limit.
 
-You can set a custom preferred height of ModalDialog using a custom property:
-
-- `preferredHeightOnMobile` for mobile screens, and
-- `preferredHeightFromTabletUp` for tablet screens and up.
+You can set a custom preferred height of ModalDialog using the `height` prop.
+The prop accepts any valid CSS value, either as a string or an object with breakpoints as keys.
 
+The custom properties fall back to the previous breakpoint using the mobile-first approach. For example, if you set
```

Comment:
Do we need to talk about CSS custom properties here? I mean it's just the internal implementation of the `web` package…